src/main/scala/cromwell/engine/CallExecutionActor.scala

```diff
@@ -31,27 +37,49 @@ class CallExecutionActor(backendCall: BackendCall) extends Actor with CromwellAc
 
   private val log = Logging(context.system, classOf[CallExecutionActor])
 
-  val tag = s"CallExecutionActor [UUID(${backendCall.workflowDescriptor.shortId}):${backendCall.key.tag}]"
+  implicit val ec = context.system.dispatcher
 
-  private def execute(executionMode: ExecutionMode): Unit = {
-    log.info(s"$tag: starting ${backendCall.call.name} for workflow ${backendCall.workflowDescriptor.shortId}")
-    // If the actual execution throws an exception, catch and wrap with a `FailedExecution`.
-    // Ideally the backend would map a command execution failure to a `FailedExecution`, but
-    // if an exception propagates out of `execute` it should not take down this actor.
-    val result = Try(executionMode.execute(backendCall)) recover { case e: Exception => FailedExecution(e, None) } get
-
-    result match {
-      case SuccessfulExecution(_, _) => log.info(s"$tag: successful execution.")
-      case AbortedExecution => log.info(s"$tag: aborted.")
-      case FailedExecution(e, returnCode) => log.error(e, s"$tag: failed execution, returnCode = $returnCode")
+  /**
+   * Schedule an `IssuePollRequest` message parameterized by `handle` to be delivered to this actor according to
+   * the schedule of the `backoff`.
+   */
+  private def scheduleNextPoll(handle: ExecutionHandle): Unit = {
+    val interval = backoff.nextBackOffMillis().millis
+    context.system.scheduler.scheduleOnce(interval) {
+      self ! IssuePollRequest(handle)
     }
+  }
+
+  private val backoff = new ExponentialBackOff.Builder()
+    .setInitialIntervalMillis(5.seconds.toMillis.toInt)
+    .setMaxIntervalMillis(30.seconds.toMillis.toInt)
+    .setMaxElapsedTimeMillis(Integer.MAX_VALUE)
+    .setMultiplier(1.1)
+    .build()
+
+  val tag = s"CallExecutionActor [UUID(${backendCall.workflowDescriptor.shortId}):${backendCall.key.tag}]"
 
-    context.parent ! CallActor.ExecutionFinished(backendCall.call, result)
-    context.stop(self)
+  /** Intended for use with `Future#onComplete`, if the `Future` completes successfully apply `successFunction`
```

Comment:
I'm still wrapping my head around the changes that we needed to do, though I can't help but think that it'd be nice if the solution could at least *look* like the `TryUtil.retryBlock` solution.  Like, encapsulating the actors, messages, and timers into an API that looks like the `TryUtil.retryBlock` API.